lib/utils/fs.go

```diff
@@ -241,15 +243,67 @@ func FSTryReadLockTimeout(ctx context.Context, filePath string, timeout time.Dur
 	return fileLock.Unlock, nil
 }
 
+// RemoveAllSecure is similar to `os.RemoveAll` however will delegate to the `RemoveSecure` implementation below
```

Comment:
```suggestion
// RemoveAllSecure is similar to `os.RemoveAll` but leverages [RemoveSecure] to delete files.
```

- the square brackets create a link
- godoc is often viewed outside the source, so avoid saying "below"

Reply:
Updated, thank you!